config/v1/0000_10_config-operator_01_infrastructure.crd.yaml

```diff
@@ -150,6 +150,86 @@ spec:
                     openstack:
                       description: OpenStack contains settings specific to the OpenStack infrastructure provider.
                       type: object
+                      properties:
+                        apiLoadBalancerType:
+                          description: apiLoadBalancerType specifies the type of loadbalancer which will be configured for the API server. Permitted values are `VRRP`, `BGP`, and no value. If no value is set, `VRRP` will be assumed.
+                          type: string
+                          enum:
+                            - ""
+                            - VRRP
+                            - BGP
+                        bgpConfiguration:
```

Comment:
I was wondering if the BGP configuration shouldn't be part of the `failureDomains`? Wouldn't it make it easier?
In reality, each domain will have its own BGP config anyway.

Reply:
It certainly could be, but from what I understood, the BGP configuration doesn't matter to anything else that consumes the failure domain configuration does it? Eg a Machine or the CCM wouldn't consume BGP configuration from here?

I did mention to @mdbooth that I see the BGP speakers more like separate resources that are referencing failure domains as they are required to exist within one, more like machines than part of the failure domain definition